pkg/jx/cmd/import.go

```diff
@@ -488,14 +488,14 @@ func (options *ImportOptions) DraftCreate() error {
 		return err
 	}
 
-	provider, err := gits.CreateProvider(options.GitServer, options.GitUserAuth, options.Git())
-	if err != nil {
-		return err
+	if options.GitUserAuth == nil {
+		userAuth := options.GitProvider.UserAuth()
+		options.GitUserAuth = &userAuth
 	}
-
+	
 	if options.Organisation == "" {
 		gitUsername := options.GitUserAuth.Username
-		options.Organisation, err = gits.GetOwner(options.BatchMode, provider, gitUsername, options.In, options.Out, options.Err)
+		options.Organisation, err = gits.GetOwner(options.BatchMode, options.GitProvider, gitUsername, options.In, options.Out, options.Err)
```

Comment:
Since options.GitProvider is for sure created before you call DraftCreate(), just use it here.